file-check/README.md

```diff
@@ -0,0 +1,38 @@
+# file-check
+
+## Description
+
+This action checks if the file exists.
+
+## Usage
+
+```yaml
+jobs:
+  file-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check file exists
+        id: file-check
+        uses: autowarefoundation/autoware-github-actions/file-check@tier4/proposal
+        with:
+          files: README.yaml,LICENSE
+          condition: or
+
+      - name: Check result
+        if: steps.file-check.outputs.exists == 'true'
+        # Only executed if one of the files exists
+        run: echo One of the files exists.
+```
+
+## Inputs
+
+| Name      | Required | Description                                                                                                  |
+| --------- | -------- | ------------------------------------------------------------------------------------------------------------ |
+| files     | true     | Comma-separated file names.                                                                                  |
+| condition | true     | Set to `and` or`or`. If more than one file is specified, this condition determines whether to report `true`. |
```

Comment:
I feel this is enough. Even if you keep the current sentence, you need to add a space between `` or`or` ``.

```suggestion
| condition | true     | The `and` or `or` condition for file existence check. |
```